### PR TITLE
Add templates for RHBA

### PR DESCRIPTION
**Pull request by jakubschwan**

### CODE REVIEW

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Comment by sutaakar** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
Test KieServerWithSmartRouterHttpScalingIntegrationTest.testConnectionBetweenDeployables failing because of changes in PR #159 we can't get service name from yet not deploy template.

**Comment by jakubschwan** [comment]
retest this please

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ *
+ * @author jschwan
+ */
+public class WorkbenchKieServerDatabasePersistentScenarioBuilderImpl implements WorkbenchKieServerDatabasePersistentScenarioBuilder {
+
+    private Map<String, String> envVariables;
+
+    public WorkbenchKieServerDatabasePersistentScenarioBuilderImpl() {
+        this.envVariables = new HashMap<String, String>();
```
Comment (jhrcek) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java]: Please initialize in filed initializer (`private Map<String, String> envVariables = new HashMap<String, String>();`) and remove the superfluous `this.` in constructor to keep consistency with other scenario builders.

**Comment by jakubschwan** [comment]
@jhrcek scenario builder updater, please take a look and merge it if there aren't any new comments.

**Comment by jhrcek** [comment]
retest this

**Comment by sutaakar** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Pull request merged**